### DISCUSSION TITLE
"https://github.com" has fingerprint "undefined" error

### DISCUSSION THREAD

**Discussion by jamhanpar** [community/community#39710] [New to GitHub]
### Select Topic Area

Question

### Body

When I try to do a "git pull origin main" command, I run into this error:

"https://github.com" has fingerprint "undefined"

Any ideas on how I can resolve this issue?

**Comment by ossanna16** [comment]
Hi there @jamhanpar and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by airtower-luna** [comment]
Please share the full command you used and its output (you can censor a private repo name), otherwise it's hard to guess what may be wrong.

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @jamhanpar, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
